outputs.tf

```diff
@@ -61,3 +65,36 @@ resource "local_file" "deploy_openstack" {
   filename        = "ansible/files/deploy-openstack.sh"
   file_permission = "0755"
 }
+
+resource "ansible_host" "control_host" {
+  name   = openstack_compute_instance_v2.ansible_control.access_ip_v4
+  groups = ["control"]
```

Comment:
nit: We normally qualify this as ansible control to distinguish it from the OpenStack controllers.